### PR TITLE
Added Dockerfile and updated the Readme.

### PR DESCRIPTION
**Pull request by tobilg**
As discussed in https://github.com/henridf/apache-spark-node/issues/14

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    rm spark-$SPARK_VERSION-bin-hadoop2.6.tgz
+
+# Set SPARK_HOME
+ENV SPARK_HOME /usr/local/spark
+
+# Set ASSEMBLY_JAR
+ENV ASSEMBLY_JAR $SPARK_HOME/lib/spark-assembly-$SPARK_VERSION-hadoop2.6.0.jar
+
+# Set native Mesos library path
+ENV MESOS_NATIVE_JAVA_LIBRARY /usr/local/lib/libmesos.so
```
Comment (henridf) [inline Dockerfile]: is this needed?

Comment (tobilg) [inline Dockerfile]: Yes if you also want to run on Mesos

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+WORKDIR $APP_BASE_PATH
+
+# Setup of the configurator
+RUN chmod +x index.js && \
+    chmod +x $APP_BASE_PATH/bin/spark-node && \
+    npm install -g node-gyp && \
+    npm install && \
+    npm run compile
+
+CMD ["node", "/app/bin/spark-node"]
```
Comment (henridf) [inline Dockerfile]: EOF newline please

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+
+# Install Node.js 5.x
+ENV NODE_VERSION v5.1.0
+RUN wget --no-check-certificate https://nodejs.org/dist/$NODE_VERSION/node-$NODE_VERSION-linux-x64.tar.gz && \
+    tar -C /usr/local --strip-components 1 -xzf node-$NODE_VERSION-linux-x64.tar.gz && \
+    rm node-$NODE_VERSION-linux-x64.tar.gz
+
+# Spark ENV vars
+ENV SPARK_VERSION_STRING spark-$SPARK_VERSION-bin-hadoop2.6
+ENV SPARK_DOWNLOAD_URL http://d3kbcqa49mib13.cloudfront.net/$SPARK_VERSION_STRING.tgz
```
Comment (henridf) [inline Dockerfile]: why not use one of the official mirrors? (e.g. the ones listed at http://www.apache.org/dyn/closer.lua/spark/spark-1.5.2/spark-1.5.2.tgz)

Comment (tobilg) [inline Dockerfile]: This is actually an official download link...

Comment (henridf) [inline Dockerfile]: can you point to the source?

Comment (tobilg) [inline Dockerfile]: Go to http://spark.apache.org/downloads.html and select "Direct Download" as download type.

Comment (henridf) [inline Dockerfile]: Ah thanks, I'd missed that.

**Comment by henridf** [comment]
I'll go ahead and merge this and then make a few changes to the readme. 
Thanks for the PR @tobilg !

**Pull request merged**